lib/ansible/modules/system/listen_ports_facts.py

```diff
@@ -0,0 +1,254 @@
+#!/usr/bin/python
+# (c) 2017, Nathan Davison <ndavison85@gmail.com>
+# GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import (absolute_import, division, print_function)
+__metaclass__ = type
+
```

Comment:
Seem to be missing the `ANSIBLE_METADATA` section

The file should contain (in this order)
1 Copyright
2 from future 
3 ANSIBLE_METADATA
4 DOCUMENTATION
5 EXAMPLES
6 RETURN
7 Python imports


http://docs.ansible.com/ansible/devel/dev_guide/developing_modules_documenting.html#ansible-metadata-block

Reply:
Thanks, looks like I had it in the wrong spot - fixed now.